packages/mui-base/src/ButtonUnstyled/useButton.types.ts

```diff
@@ -39,3 +40,33 @@ export interface UseButtonParameters {
    */
   type?: React.ButtonHTMLAttributes<HTMLButtonElement>['type'];
 }
+
+export interface UseButtonReturnValue {
```

Comment:
Explicit type added for the `useButton` return value so that we can test out the generation of the API.